test/flycheck-test.el

```diff
@@ -3463,27 +3463,29 @@ Why not:
    "language/less/syntax-error.less" 'less-css-mode
    '(1 1 error "Unrecognised input" :checker less)))
 
-(flycheck-ert-def-checker-test luacheck lua syntax-error
+(flycheck-ert-def-checker-test lua-luacheck lua syntax-error
   (flycheck-ert-should-syntax-check
    "language/lua/syntax-error.lua" 'lua-mode
-   '(5 7 error "unfinished string" :id "E011" :checker luacheck)))
+   '(5 7 error "unfinished string" :id "E011" :checker lua-luacheck)))
 
-(flycheck-ert-def-checker-test luacheck lua warnings
+(flycheck-ert-def-checker-test lua-luacheck lua warnings
   (flycheck-ert-should-syntax-check
    "language/lua/warnings.lua" 'lua-mode
    '(1 1 warning "setting non-standard global variable 'global_var'"
-       :id "W111" :checker luacheck)
+       :id "W111" :checker lua-luacheck)
    '(3 16 warning "unused function 'test'"
-       :id "W211" :checker luacheck)
+       :id "W211" :checker lua-luacheck)
    '(3 21 warning "unused argument 'arg'"
-       :id "W212" :checker luacheck)
+       :id "W212" :checker lua-luacheck)
    '(4 11 warning "variable 'var2' is never set"
-       :id "W221" :checker luacheck)))
+       :id "W221" :checker lua-luacheck)))
 
 (flycheck-ert-def-checker-test lua-luacheck lua no-warnings
-  (let ((flycheck-luacheckrc "luacheckrc"))
-    (flycheck-ert-should-syntax-check
-     "language/lua/warnings.lua" 'lua-mode)))
+  (copy-file (flycheck-ert-resource-filename "language/lua/luacheckrc")
+             (flycheck-ert-resource-filename "language/lua/.luacheckrc"))
+  (flycheck-ert-should-syntax-check
+   "language/lua/warnings.lua" 'lua-mode)
+  (delete-file (flycheck-ert-resource-filename "language/lua/.luacheckrc")))
```

Comment:
That's a nice idea, but as far as I'm concerned I'd like to ask you to remove the test case entirely.  All that we're still testing here is that luacheck itself works correctly and I think that's not our concern.  I think we can rely upon luacheck to do its job correctly.

Please remove this test case and the `luacheckrc` configuration file.  Let's keep the tests confined to what's really done by Flycheck, i.e. parsing the output of Luacheck which is already covered by the other test cases for luacheck.